CS3219-AY2223S1/cs3219-project-ay2223s1-g12
# Implement backend validation for password strength

Fixes #79 
Depends on #67 

Summary:
- Adds API validation for password strength: At least 8 characters, and is alphanumeric.
- Further style error and success snackbars by using MUI alerts

How to test:
1. In `user-service`, run `npm run dev`
2. In postman, try creating a user with weak password strength like `cs3219` (too short). You should receive an error message `New password does not meet password strength requirement. Passwords should contain at least 8 characters and is a combination of numbers and alphabets `
3. In postman, when changing password, try using old password as new password too. You will receive an error message too.
4. Otherwise, enter a strong password, and you will receive a success message.
